indico/htdocs/js/indico/RegistrationForm/section.js

```diff
@@ -705,8 +705,9 @@ ndRegForm.directive("ndSocialEventSection", function() {
                            align: 'center',
                            width: 50,
                            editable: true,
-                           edittype: "text",
+                           edittype: 'text',
                            editoptions: {size: "7", maxlength: "20"},
+                           pattern: '/^(\\d*[.])?\\d+$/'
```

Comment:
You can't rule out such possiblity for sure. To me it is just a convenience, not worth a bigger discussion.